factor/sms/classes/event/sms_sent.php

```diff
@@ -0,0 +1,44 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+/**
+ * Event for a sent SMS
+ *
+ * @package     factor_sms
+ * @author      Alex Morris <alex.morris@catalyst.net.nz>
+ * @copyright   Catalyst IT
+ * @license     http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+
+namespace factor_sms\event;
+
+defined('MOODLE_INTERNAL') || die();
+
+class sms_sent extends \core\event\base {
+
+    protected function init() {
+        $this->data['crud'] = 'r';
+        $this->data['edulevel'] = self::LEVEL_OTHER;
+    }
+
+    public function get_description() {
+        return "The user with id '{$this->other['userid']}' had an SMS message sent to them. <br> Information: {$this->other['debug']}";
```

Comment:
maybe this should be a tiny bit more specific to MFA, eg `had a verification code sent to the via SMS` or similar